### ISSUE TITLE
Sprint #5 Retrospective

### ISSUE THREAD

**Issue by qspraut** [dartmouth-cs98/21f-macro-meals#237]
## Sprint #5 Retrospective:
* members in attendance:
  ![selfie proof of meeting](https://imgur.com/ZDztX1b.jpg)
  - [x] Alex
  - [x] Anand
  - [x] Brandon
  - [x] Quinn
* what worked well
  * Debugged some issues with new classification model and found more training images
  * added spoonacular integration can now query recipes filtering with macro and 
  * Added images to training set and re-trained model
  * Debugged model's training method; re-factored to allow for easier training in the future 
  * Added favorite feature to previous meals
  * Finished community tab - favorites and top rated tabs now working
  * Added ability to make meals private
  * Implemented new uniform styling throughout app
  * Refactored some ugly code - condensed & reorganized
  * Added constraints to model (e.g an apple shouldn't be 1000+ calories)
* what didn't
  * New model was not implementable due to deprecated structures, so we decided to further train previous model
  * Re-trained model showed lower accuracy, though this is probably due to test images being taken from the old training set
  * Still struggling with iOS bug - going to bring focus back onto this this week after a bit of a break
* self assessment on progress
  * where are you in relation to progress towards product and milestones?
    * Approximately 85% done overall (70% done with progress for this term)
    * We feel good about current location - we have some decent improvement in core features and stretch features are being implemented
    * lay out *each* of the following weeks till end of term with brief goals for each
      * Current week: Implement confidence scores, test app with new users and have them interact with community, continue making model improvements and increase training data, add food journaling feature, UI representation of the spoonacular stuff, FIX IOS BUG (new camera package possibly)
      * week 7: Finish stretch features for food recommendations based on goals and history
      * week 8: Write medium article and market copy to tech community
      * week 9: Make final adjustments and test with more users
* briefly summarize any other topics/discussions
    * happy with addition of more features, happy with the increased functionality slowly transforming app into a more robust meal and macro tracker (community tab and spoonacular)
    * Need to get ML server back up and running effectively for live demos
